.github/workflows/project-management.yml

```diff
@@ -0,0 +1,21 @@
+name: Project Management
+
+on:
+  schedule:
+    - cron: '*/5 * * * *'
+
+jobs:
+  # Best used in combination with actions/stale to assign a Stale label
+
+  unassign-issues-labeled-waiting-for-contributor-after-1-days-of-inactivity:
+    name: Unassign issues labeled "Stale" after 1 days of inactivity.
+    if: github.event_name == 'schedule'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: boundfoxstudios/action-unassign-contributor-after-days-of-inactivity@main
+        with:
+          last-activity: 1 # After how many days the issue should get unassigned
+          labels: 'Stale' # Only search for issues containing this labels (comma-separated)
```

Comment:
can we search for all "help wanted" labelled issues instead please

Reply:
> can we search for all "help wanted" labelled issues instead please

Noted.

Reply:
Updated